### PR TITLE
Correct instructions for enabling the firewall.

### LINKED ISSUE #7294: Configuring the firewall - installing ufw disables all ports - unable to ssh
This just creates so much problem as new users will just install ufw and now its impossible to access the machine. Port 22 is permanently disabled and ssh is not possible at all. 

Even adding a firewall rule on google console does not help at all.

I have no choice but to delete the VM and install it again. UFW now rejects all traffic on ports except 80 and 443.

Documentation should add some warning sign or update the docs on what all can go wrong when ports are closed.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dcdd2c35-bf2b-2673-d150-303a8aaa54d2
* Version Independent ID: 815fe034-c9fc-aee3-a8b5-f2e860b30302
* Content: [Host ASP.NET Core on Linux with Nginx](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?view=aspnetcore-2.1&tabs=aspnetcore2x)
* Content Source: [aspnetcore/host-and-deploy/linux-nginx.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/linux-nginx.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by StillLearnin**
Fixes #7294

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@StillLearnin  can you review the changes in my commit?

**Comment by StillLearnin** [comment]
Hmm... I'm ok with more concise wording but it is fairly easy to miss and most VPS hosts don't use the default port 22. Is there a way to make it stand out that you **will LOSE access** to your machine if you're not careful without using a warning?

**Comment by Rick-Anderson** [comment]
@StillLearnin  Ah, I missed that point. Reverted.

**Comment by guardrex** [comment]
Additional remark: https://github.com/aspnet/Docs/issues/7294#issuecomment-407304444

**Comment by StillLearnin** [comment]
I was debating with myself on how to respond to the comments. I think that adding the links would probably be a good idea. Taking out the firewall section or recommending skipping it isn't wise in my opinion. The last thing we need it more bots added to botnets. Even if the programmer doesn't fully understand firewalls, the basic instructions as we have them should enable them to reduce the hackability of their machine. Links to additional resources would be a nice touch. After all, the reason a programmer is reading the docs is because he doesn't know the information already.

**Comment by StillLearnin** [comment]
@Rick-Anderson go ahead and add the links.

**Comment by Rick-Anderson** [comment]
@StillLearnin @shirhatti  please review.

**Comment by StillLearnin** [comment]
@Rick-Anderson Ok, I'm satisfied.
